python_dependencies/processing/requirements.txt

```
@@ -1,8 +1,8 @@
-anndata==0.10.8
+anndata==0.11.2
 awscli
 boto3>=1.11.17
-botocore>=1.14.17
-cellxgene-schema
+git+https://github.com/chanzuckerberg/single-cell-curation/@c6471f8435f01bc78a7b6084aa1d2464645d5554#subdirectory=cellxgene_schema_cli
```

change back before merging to main.
```suggestion
cellxgene-schema
```